### ISSUE TITLE
Baseline: AutoEncoder

### ISSUE THREAD

**Issue by porterjenkins** [porterjenkins/region-representation-learning#13]
We need to establish baseline methods to test our model against. One obvious baseline is the AutoEncoder using the just the satellite images.

1. Train an encoder (`AutoEncoder.py`) using the same loss and hyperparameters as our proposed method
2. Extract the hidden representation. This should be automatically written to `embedding` directory.
3. Import the hidden representation to the file `experiments/run_predict_job.py.`

- Use hidden representation as features in an xgboost model. 

- Evaluate k-fold CV error

**Comment by porterjenkins** [comment]
I met with my advisor this morning. She really wanted to see the AutoEncoder and PCA baselines. I went ahead and did them. Would love if you could help with the Tile2Vec tho :)
